static/js/components/widgets/ResourcePickerDialog.test.tsx

```diff
@@ -72,7 +72,7 @@ describe("ResourcePickerDialog", () => {
       ),
       {
         mode:         RESOURCE_EMBED,
-        contentNames: ["resource", "pages"],
+        contentNames: ["resource", "page"],
```

Comment:
So I believe what was happening was...

1. `contentNames` should be `CollectionName[]`
2. `<ResourcePickerDialog />` filters the contentNames based on what collections actually exist in the config

In the test file, there was a mismatch between `contentNames` and the actual collection names.

Note:
    - in the test file, `contentNames` is passed manually
    - in "real" ocw, `contentNames` is calculated in a parent component, `<MarkdownEditor />` based on the config

The calculation of `contentNames` could be moved into ResourcePickerDialog (nothing else uses it) then the prop could be gotten rid of all together. (It'd still be possible to have a mismatch in the config, though).